src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_rpc_plugin.cc

```diff
@@ -61,17 +64,11 @@ RCRPCPlugin::Apps RCRPCPlugin::GetRCApplications(
   using application_manager::ApplicationSet;
   ApplicationSet accessor = app_mngr.applications().GetData();
 
-  auto predicate = [](const ApplicationSharedPtr& app) {
-    auto uid = RCRPCPlugin::kRCPluginID;
-    return app ? app->QueryInterface(uid).valid() : false;
-  };
-
-  auto it = std::find_if(accessor.begin(), accessor.end(), predicate);
-
   std::vector<ApplicationSharedPtr> result;
-  while (it != accessor.end()) {
-    result.push_back(*it);
-    it = std::find_if(++it, accessor.end(), predicate);
+  for (auto& it : accessor) {
```

Comment:
@BSolonenko Why not `const`?

Reply:
@AByzhynar  fixed in https://github.com/smartdevicelink/sdl_core/pull/2048/commits/3ffd281451b051e9ead29c3a877a6678109321ac